test/evergreen.yml

```diff
@@ -3841,6 +3884,7 @@ tasks:
           timeout_secs: 86400
       - func: "fetch mongo repo"
       - func: "get project"
+      - func: "get engflow creds"
       - func: "import wiredtiger into mongo"
       - func: "compile mongodb"
```

Comment:
`compile mongodb` is passing `ICECC=icecc CCACHE=ccache` into the scons call. 
Will this line be changed as part of future DEVPROD-257 work, or should the WiredTiger team keep an eye on this and update it ourselves?